docs/using-radicle/tracking-and-viewing.md

```diff
@@ -0,0 +1,40 @@
+---
+id: tracking-and-viewing
+title: Tracking and viewing contributions
+---
+
+To view someone else's changes to your project, you first need to add them as a
+`remote` to track their view of your project.
+
+### Adding remotes
+To view and pull changes from another user's version of your project, you must
+first **add them as a remote**. To do this, you must add their **Device ID** to
+the list of remotes that you follow. You can do so by clicking Manage Remotes in
+the peer switcher on your project page and adding their Device ID. This is also
```

Comment:
Are we calling it "peer switch"? :joy:

Reply:
I guess? That's what @xla called it.

Reply:
I believe we call it "peer selector" :smile:

Reply:
oops

Reply:
```suggestion
the peer selector on your project page and adding their Device ID. This is also
```